packages/tests-e2e/e2e-tests/categories.cy.mock.ts

```diff
@@ -0,0 +1,20 @@
+import { mock } from "mocks/factory";
+
+describe(`when I visit the "Categories" page`, () => {
+  const mockCategories = mock.categories(6);
```

Comment:
🤔 In the project I am working on, I've included my mock and real tests in the same file, with `realOnly(() => {})` and `mocksOnly(() => {})` closures.  The reason for this was that sometimes a significant portion of the tests are identical for each data source (visit this, click that, fill these fields in), and all that will sometimes change are the values being used for acting and asserting.

I'm not specifically wedded to it, and I can see the benefits and purity of your separate file approach; I just wondered whether you had encountered much duplication in both flavours of tests?

Reply:
I could definitely see the benefit in your approach. The "real" and "mock" tests have a ton in common for sure.
In this codebase, I realized that basically all of my "real" tests are totally compatible with the mock server.
However, I would rather have in-file differences like you suggest

Reply:
I made this change in #155 ! Thanks for the suggestion!